lib/counter_culture/reconciler.rb

```diff
@@ -101,12 +102,23 @@ def perform
             # now iterate over all the models and see whether their counts are right
             update_count_for_batch(column_name, records)
           end
+          log << "\n" if log.info?
         end
       end
 
       private
 
+      def log
+        @log ||= if options[:verbose]
+                   Logger.new(STDOUT)
+                 else
+                   Logger.new('/dev/null')
```

Comment:
I don't love this; some thoughts / questions:
1. Why not use `Rails.logger` here, at least if it's available?
2. I wouldn't use a logger to `/dev/null`, I think I'd prefer us to encapsulate this with a `log(message)` method that logs only if verbose is true.

What do you think?